gltf/gltfpack.cpp

```diff
@@ -790,8 +838,12 @@ int gltfpack(const char* input, const char* output, const char* report, Settings
 
 	const char* iext = strrchr(input, '.');
 	const char* oext = output ? strrchr(output, '.') : NULL;
+	
+	if (iext && (strcmp(iext, ".vrm") == 0 || strcmp(iext, ".VRM") == 0)) {
```

Comment:
Please use clang-format on this code, as this bracing style is inconsistent with the coding guidelines, see https://github.com/zeux/meshoptimizer/blob/master/CONTRIBUTING.md